src/report.sh

```diff
@@ -350,9 +350,15 @@ function save_data_to()
     exit "$ret"
   fi
 
-  truncate -s 0 "$path"
-  [[ -n "${options_values['POMODORO']}" ]] && show_data "$target_time" >> "$path"
-  [[ -n "${options_values['STATISTICS']}" ]] && show_statistics >> "$path"
+  if [[ -d "$path" ]]; then
+    
```

Comment:
You need to remove the whitespace from this line. https://github.com/kworkflow/kworkflow/actions/runs/3260634561/jobs/5354332619